reusable_workflows/check_membership/check_membership/action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
     required: true
   user:
     description: "The username"
-    default: ${{ github.event.pull_request.user.login }}
+    default: ${{ github.actor }}
```

Comment:
Will that be actually what we want?
If an external user opened a PR, and then an internal user started the pipeline, who will be github.actor?

Reply:
Ah very good point - yes, I'll change this line back 👍